opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -113,6 +138,17 @@
          */
         SpanBuilder addReference(String referenceType, SpanContext referencedContext);
 
+        /**
+         * Remove any explicit (e.g., via {@link SpanBuilder#addReference(String,SpanContext)}) or implicit (e.g., via
+         * {@link ActiveSpanSource#activeContext()}) references to parent / predecessor SpanContexts, thus making the built
+         * Span a "root" of a Trace tree/graph.
```

Comment:
@yurishkuro IMO this would be misnamed in that case... could be `SpanBuilder ignoringActiveSpan()`, maybe? If we want to force a root per `asRoot`, IMO the semantics described by the comment are appropriate.